AppServer_Java/src/com/google/appengine/tools/development/DevAppServerImpl.java

```diff
@@ -190,6 +190,13 @@ public void run() {
     else
     {
         System.setProperty(APPLICATION_ID_PROPERTY, config.getAppId());
+        // AppScale: Set MODULE and VERSION env variables for taskqueue.
+        System.setProperty("MODULE", mainModule.getModuleName());
+        String versionId = config.getMajorVersionId();
+        if (versionId == null)
+          versionId = "v1";
```

Comment:
although I suspect that with the changes that were made to the devappserver it might not be necessary to have this if statement because I think version is always set to v1... I'll test that